js/archive.js

```diff
@@ -11,6 +11,15 @@
 
   // this runs after the DOM has loaded
   $(function() {
+    try {
+      if (!window.location.search) return;
+      const { search: params } = window.location
```

Comment:
Destructuring assignment won't work in IE 

https://caniuse.com/#search=destructuring%20assignment